fault-tolerance/src/main/java/module-info.java

```diff
@@ -20,7 +20,8 @@
 module io.helidon.faulttolerance {
     requires io.helidon.config;
     requires io.helidon.common.configurable;
+    requires io.helidon.common.reactive;
```

Comment:
This should be a transitive dependency, as Single and Multi are part of public API of this module.

Reply:
Fixed